frontend/src/components/Login.jsx

```diff
@@ -26,15 +26,12 @@ class Login extends Component {
       location
     } = response;
 
-    console.log("RESPONSE", accounts.data[0]);
-
     const user = {
       firstName: first_name,
       lastName: last_name,
       email,
       facebookId: id,
-      instagramId: accounts.data[0].id,
-      imageUrl: picture.data.url,
+      instagramId: accounts.data[0].instagram_business_account.id,
```

Comment:
@effyyzhang this is causing an error because we no longer have instagram_business_account inside accounts.data

Reply:
? did we merge any changes?

Reply:
@clacayetano  Checked again, I didn't ran into any error. I added the instagram_business_account back cause the previous cold was getting the wrong id, the previous code is getting the id of the instagram_business_account object, once I add it back, we are getting the instagram_busines_account's property.